.github/workflows/test_openvino.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.8, 3.9]
```

Comment:
Is it possible to also test with Python 3.9 (and 3.10 once that is supported)?

Reply:
Sure let me add it